pkgs/tools/security/browserpass/default.nix

```diff
@@ -1,18 +1,20 @@
 { lib, buildGoModule, fetchFromGitHub, makeWrapper, gnupg }:
 buildGoModule rec {
   pname = "browserpass";
-  version = "3.0.6";
+  version = "3.0.10";
 
   src = fetchFromGitHub {
     owner = "browserpass";
     repo = "browserpass-native";
     rev = version;
-    sha256 = "0q3bsla07zjl6i69nj1axbkg2ia89pvh0jg6nlqgbm2kpzzbn0pz";
+    sha256 = "8eAwUwcRTnhVDkQc3HsvTP0TqC4LfVrUelxdbJxe9t0=";
   };
 
   nativeBuildInputs = [ makeWrapper ];
 
-  vendorSha256 = "1wcbn0ip596f2dp68y6jmxgv20l0dgrcxg5cwclkawigj05416zj";
+  vendorSha256 = "gWXcYyIp86b/Pn6vj7qBj/VZS9rTr4weVw0YWmg+36c=";
+
+  CGO_ENABLED = 0;
```

Comment:
Is this not redundant since upstream's Makefile sets it to 0 too? I could be wrong I haven't tested.

Reply:
Hmm, you might be right. This has been added in the Makefile in version 3.0.9. So this should not be needed. Let me push a change tomorrow to remove it, so that we keep the build logic fully in the Makefile! Thanks for spotting that.

Reply:
I have just pushed to remove that.